src/cli/args.zig

```diff
@@ -200,7 +203,7 @@ fn formatInvalidValue(
     var buf = std.ArrayList(u8).init(arena_alloc);
     errdefer buf.deinit();
     const writer = buf.writer();
-    try writer.print("invalid value \"{?s}\"", .{value});
+    try writer.print("{s} \"{?s}\"", .{ i18n._("invalid value"), value });
```

Comment:
This is a bad idea for multiple reasons:
- Not all languages use English/Latin alphabet quotes
- Not all languages place the string as a postfix
- Not all languages even have "invalid value" and the string as two separate, distinct parts

And so on and so forth. As Mitchell stated we should explore sprintf-style printing first, or implement another mechanism for substituting variables